CHANGELOG.md

```diff
@@ -23,6 +23,14 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/virtual-users/compare/release-3.9.1...master
 
+### Added
+- Add `TimeControllingUserGenerator` for synchronizing user generators across threads and nodes.
```

Comment:
TBH it's fully implementable by users, so we don't have to provide it right now. I can cut it out of the PR.